streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/ClientState.java

```diff
@@ -50,10 +49,17 @@
     private final Set<TaskId> prevStandbyTasks;
 
     private final Map<String, Set<TaskId>> consumerToPreviousStatefulTaskIds;
+    // the following four maps are used only for logging purposes;
+    // TODO: we could consider merging them with other book-keeping maps
```

Comment:
Could we add a follow-up ticket instead? Someone in the community could pick it up.